typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -418,10 +421,15 @@ class ContextFunder {
       desiredBalance,
       'ether',
     );
+    const adjustedDesiredBalance =
+      this.context === Contexts.ReleaseCandidate
+        ? desiredBalanceEther.mul(RC_FUNDING_DISCOUNT)
```

Comment:
This'll throw
```
> n = ethers.BigNumber.from(100)
BigNumber { _hex: '0x64', _isBigNumber: true }
> n.mul(0.2)
Uncaught:
Error: underflow [ See: https://links.ethers.org/v5-errors-NUMERIC_FAULT-underflow ] (fault="underflow", operation="BigNumber.from", value=0.2, code=NUMERIC_FAULT, version=bignumber/5.7.0)
```

Something like the NUMERATOR / DENOMINATOR thing we do for the min delta would work well here

Reply:
Great catch!